core_api_c_binding/src/lib.rs

```diff
@@ -3,7 +3,7 @@ extern crate holochain_core;
 extern crate holochain_core_api;
 extern crate holochain_dna;
 
-use holochain_core::context::Context;
+use holochain_core::context::{Context, STORAGE_PATH};
```

Comment:
This is weird.

The storage path should not be in the module context, and not in the crate core.
Actually the storage path has to come in as a parameter to `holochain_new` below.